000-ml-project-discovery.md

```diff
@@ -0,0 +1,81 @@
+- Feature Name: ml_project_discovery
+- Start Date: 2020-10-21
+- RFC PR: [amundsen-io/rfcs#000-ml-project-discovery.md](https://github.com/amundsen-io/rfcs/pull/0000) (after opening the RFC PR, update this with a link to it and update the file name)
+- Amundsen Issue: [amundsen-io/amundsen#0000](https://github.com/amundsen-io/amundsen/issues/0000) (leave this empty for now)
+
+# ML project discovery using Amundsen
+
+## Summary
+
+Our proposal is to extend the capabilities of Amundsen by introducing new data resources covering the ML project lifecycle.
+
+## Motivation
+
+The rising number of ML projects force organizations to address challenges related to model governance and model discovery.
+According to latest market surveys one can expect significant acceleration in AI adoption this and next year.
+
+## Guide-level Explanation (aka Product Details)
+
+The concept of machine learning project refers to complex process of moving your research from local exploration environment towards production grade deployments.
+Essentially, that consists of multiple steps including feature engineering, tracking experiments, model training, batch scoring, model versioning, model staging, release and deployment.
+Many organizations suffers from lack of visiblity over the ML assets delivered by the different teams of data scientists.
+Such model catalog would improve the collaboration within the organization through single interface to discover available ML resources and related metadata.  
+
+The basic use cases can be described as follow:
+
+*   search for ml projects by project name, project description, project owner
+*   search for ml models by model name, model description, model owner, model stage
+*   search for model deployments by deployment name, deployment description, deployment sample request/response (prediction)
```

Comment:
@dwarszawski does it mean introducing 3 additional entities on top of tables and dashboards ?

Reply:
it's continuously evolving, I think it's better to describe it in terms of the metadata available during different stages of ml project lifecycle.